src/EditorFeatures/VisualBasic/LineCommit/ICommitFormatter.vb

```diff
@@ -8,18 +8,22 @@ Imports Microsoft.VisualStudio.Text
 Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
     Friend Interface ICommitFormatter
         ''' <summary>
-        ''' Commits a region by formatting and case correcting it. It is assumed that an
-        ''' ITextUndoTransaction is open the underlying text buffer, as multiple edits may be done
-        ''' by this function. Further, if the operation is cancelled, the buffer may be left in a
-        ''' partially committed state that must be rolled back by the transaction.
+        ''' Commits a region by formatting and case correcting it. It is assumed that an ITextUndoTransaction is open
+        ''' the underlying text buffer, as multiple edits may be done by this function. Further, if the operation is
+        ''' cancelled, the buffer may be left in a partially committed state that must be rolled back by the
+        ''' transaction.
```

Comment:
So if an undo transaction is left open, what is happening to the other editor edits? I would have presumed those are getting included in the paste format transaction, or other bad things might happen.

(I know the editor team has been tracking down issues with transactions being left open while the user can type and I think that's very bad.)